api/src/org/labkey/api/util/ExceptionUtil.java

```diff
@@ -896,12 +898,16 @@ private static void renderErrorPage(Throwable ex, int responseStatus, String mes
 
             addDependenciesAndRender(responseStatus, pageConfig, errorView, ex, request, response);
         }
+        catch (ConfigurationException ce)
+        {
+            throw new ConfigurationException(ce.getMessage());

```

Comment:
Just have this throw the ce directly - no need to create a new exception (and this current code will lose the original stack trace).

```suggestion
            throw ce;
```